riff-raff/app/ci/S3Build.scala

```diff
@@ -38,6 +43,18 @@ class S3BuildOps(config: Config) extends Logging {
     Nil
   } get
 
+  private def buildMarker(build: S3Build): MarkerContext = {
+    val params: Map[String, String] = Map(
+      "projectName" -> build.jobName,
+      "projectBuildNumber" -> build.number,
+      "projectBuildTool" -> build.buildTool.getOrElse("unknown"),
+      "projectVcsUrl" -> VCSInfo.normalise(build.vcsURL).getOrElse("unknown"),
+      "projectVcsBranch" -> build.branchName,
+      "projectVcsRevision" -> build.revision
```

Comment:
Prefixing these with `project` to match the [GrafanaAnnotationLogger](https://github.com/guardian/riff-raff/blob/16b4cd79c7dda073cb57fe987b7dee470f33b4fb/riff-raff/app/notification/GrafanaAnnotationLogger.scala#L35-L44).